www/docs/src/content/docs/contributing/translations.mdx

```diff
@@ -0,0 +1,30 @@
+---
+title: Translations
+description: Learn how to help translate StudioCMS
+sidebar:
+  order: 2
+---
+
+import ReadMore from '~/components/ReadMore.astro';
+
+StudioCMS is a global project, and we want to make it accessible to everyone. If you are fluent in multiple languages, you can help us translate StudioCMS into your language.
+
+## Dashboard i18n
+
+Current translation status:
+
+<img src="https://i18n.studiocms.xyz/widget/studiocms/horizontal-auto.svg" alt="Translation status" />
+
+Visit [our i18n dashboard](https://i18n.studiocms.xyz) to help translate StudioCMS into your language. If your language is not listed, you can add it within the dashboard.
+
+If you would prefer to contribute translations directly to the repository, the translations are stored in the [`packages/studiocms_core/src/i18n/translations`](https://github.com/astrolicious/studiocms/tree/main/packages/studiocms_core/src/i18n/translations/) directory. You can find the English translations in the [`en-us.json`](https://github.com/astrolicious/studiocms/blob/main/packages/studiocms_core/src/i18n/translations/en-us.json) file.
+
+<ReadMore>
+StudioCMS uses [Weblate](https://weblate.org) for managing translations on top of GitHub. If you are new to Weblate, you can find the [Translating using Weblate Guide](https://docs.weblate.org/en/latest/user/translating.html#) on their website.
+</ReadMore>
+
+Once the translations have been added, they will be added to the [StudioCMS i18n configuration](https://github.com/astrolicious/studiocms/blob/main/packages/studiocms_core/src/i18n/index.ts#L8) and will be available in the next release.
```

Comment:
Does the linked code snippet not refer to adding a language rather than translations? Could also remove the link and say something like:
> Once the translations have been created and approved, they will become available in the next release.

Reply:
The translation file might exist by the automatic system(Weblate) but may not yet be in the i18n confiuration(The only part of manual config we have) to be available to end-users